### PR TITLE
Git flow update

### PR DESCRIPTION
**Pull request by atenaiis**
Team,

Thank you for you comments ,

I worked on correcting Git flow.

Cheers

Mariana A. Campos

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

### Hello @atenaiis, you have done a great job so far, your code is good and I like your readme file :+1: but please implement the rubocop issue:
- [x] Rubocop is failing locally and currently you have 2 offenses, kindly fix them

Please fix the required changes and ask for another code review whenever you're ready

_**If you have any question related to the required changes feel free to text me on slack @GzimAsani** or just add a comment here_

HAPPY CODING :computer: :desktop_computer:

**Comment by atenaiis** [comment]
Thanks for your comments.
I ran rubocop on my terminal to check the offences and fix them.
Cheers.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,78 @@
 # README
```
Comment (enaburekhan) [inline README.md:1]: Kindly rename `README` to the project title: Micro-Reddit

File: README.md [hunk 2]
```diff
@@ -1,24 +1,78 @@
...
 
-* System dependencies
+## Setup
 
-* Configuration
+- To run it in your local machine you need to follow these steps.
 
-* Database creation
+## Clone
+ - Clone repository repository with this command on terminal
```
Comment (enaburekhan) [inline README.md:16]: Kindly remove one of the `repository`.

**Review by enaburekhan — APPROVED** [review]
# Status: Approve:heavy_check_mark:
**Hi Atenais!
Congratulations on this project and thank you for implementing the changes requested by the last reviewer:clap::clap:. 
As highlighted by the last reviewer, the readme is very descriptive. Keep up the great job:muscle:.**
I will encourage you to fix the optional change below. And then go ahead and merge the PR.

 [Optional ]
- Kindly check my two comments on the readme below in the PR. Fix them before you merge.

For questions/comments, please drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy coding:blush:
Eric Enaburekhan

**Pull request merged**